couchdbkit/version.py

```diff
@@ -4,6 +4,5 @@
 # See the NOTICE for more information.
 
 from __future__ import absolute_import
-from six.moves import map
```

Comment:
Curious, what does this fix?

Reply:
It errors if six isn't installed before couchdbkit-jsonobject